.github/CHANGELOG.md

```diff
@@ -98,11 +98,42 @@
   [(#270)](https://github.com/XanaduAI/strawberryfields/pull/270)
 
 * Adds support in the applications layer for node-weighted graphs. Users can sample from graphs
-  with node weights using the WAW encoding and input node weights into search algorithms in the
-  `clique` and `subgraph` modules.
+  with node weights using the WAW encoding that rescales an adjacency matrix `A` to `WAW` with `W`
+  a diagonal matrix of node weights:
   [(#295)](https://github.com/XanaduAI/strawberryfields/pull/295)
+   
+  ```python
+  from strawberryfields.apps import sample
+  import networkx as nx
+  import numpy as np
+
+  np.random.seed(1968)
+
+  g = nx.erdos_renyi_graph(20, 0.6, seed=1968)
+  a = nx.to_numpy_array(g)
+  w = [i for i in range(20)]
+
+  a = sample.waw_matrix(a, w)
+  s = sample.sample(a, n_mean=10, n_samples=10)
+
+  s = sample.postselect(s, min_count=4, max_count=20)
+  s = sample.to_subgraphs(s, g)
+  ```
+  
+  Node weights can be input to search algorithms in the `clique` and `subgraph` modules:
```

Comment:
Is this missing a bullet point?

Reply:
No, it's a continuation from the previous.

Think it looks ok though:
![image](https://user-images.githubusercontent.com/49409390/76972389-e7ebd700-6904-11ea-99e2-afb6619d5b3e.png)